Hanna/app-hw/my-app/src/hw38/PostList/Posts/PostList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Post } from '../Post/Post'
+
+export class PostList extends React.Component {
+    render() {
+        return (
+            <div className='posts'>
+                <Post post={{
```

Comment:
prefer to move data away from the component.
to separate file, or const outside `PostList`

Reply:
done (i hope)